shared/getting-started/getDeviceOnDash/artik530.md

```diff
@@ -1,12 +1,12 @@
-In order to get resinOS up and running on your **{{ $device.name }}**, you need to first set it up to boot from your {{ $device.bootMedia }} rather than its internal [eMMC memory][emmc-link]. To do this we need to set the `SW402` dip switches to position `1:off, 2:off, 3:off, 4:on`.
+In order to get balenaOS up and running on your **{{ $device.name }}**, you need to first set it up to boot from your {{ $device.bootMedia }} rather than its internal [eMMC memory][emmc-link]. To do this we need to set the `SW402` dip switches to position `1:off, 2:off, 3:off, 4:on`.
```

Comment:
```suggestion
In order to get{{ $names.os.lower }} up and running on your **{{ $device.name }}**, you need to first set it up to boot from your {{ $device.bootMedia }} rather than its internal [eMMC memory][emmc-link]. To do this we need to set the `SW402` dip switches to position `1:off, 2:off, 3:off, 4:on`.
```